magstock/__init__.py

```diff
@@ -45,6 +52,10 @@ class Attendee:
     def food_cost(self):
         return c.FOOD_PRICE if self.purchased_food else 0
 
+    @property
+    def auto_food(self):
+        return self.badge_type in [c.STAFF_BADGE, c.GUEST_BADGE]
```

Comment:
Hm, I notice that even though this property is here, it's defined a second time in the SessionMixin class. That could cause problems later if one is updated but not the other.

It's probably not a big deal since this is just the plugin for Magstock, though.